build failure: several bluetooth samples fail to build on nrf51dk_nrf51422

If I do:

```
./scripts/sanitycheck -p nrf51dk_nrf51422 -T samples/bluetooth/
```

I get:
```
ERROR   - nrf51dk_nrf51422          samples/bluetooth/hci_spi/sample.bluetooth.hci_spi FAILED: Build failure
ERROR   - see: /home/galak/git/zephyr/sanity-out/nrf51dk_nrf51422/samples/bluetooth/hci_spi/sample.bluetooth.hci_spi/build.log
INFO    - Total complete:    2/   4  50%  skipped:    0, failed:    1
ERROR   - nrf51dk_nrf51422          samples/bluetooth/hci_uart/sample.bluetooth.hci_uart.nrf5 FAILED: Build failure
ERROR   - see: /home/galak/git/zephyr/sanity-out/nrf51dk_nrf51422/samples/bluetooth/hci_uart/sample.bluetooth.hci_uart.nrf5/build.log
INFO    - Total complete:    3/   4  75%  skipped:    0, failed:    2
ERROR   - nrf51dk_nrf51422          samples/bluetooth/hci_uart/sample.bluetooth.hci_uart.arm FAILED: Build failure
ERROR   - see: /home/galak/git/zephyr/sanity-out/nrf51dk_nrf51422/samples/bluetooth/hci_uart/sample.bluetooth.hci_uart.arm/build.log
```

looking in logs:
```
/home/galak/git/zephyr/subsys/bluetooth/controller/ll_sw/ull_conn.c: In function 'force_md_cnt_calc':
/home/galak/git/zephyr/subsys/bluetooth/controller/ll_sw/ull_conn.c:6330:19: error: 'CONFIG_BT_CTLR_DATA_LENGTH_MAX' undeclared (first use in this function)
 6330 |  time_incoming = (CONFIG_BT_CTLR_DATA_LENGTH_MAX << 3) *
      |                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/galak/git/zephyr/subsys/bluetooth/controller/ll_sw/ull_conn.c:6330:19: note: each undeclared identifier is reported only once for each function it appears in
gmake[2]: *** [zephyr/subsys/bluetooth/controller/CMakeFiles/subsys__bluetooth__controller.dir/build.make:252: zephyr/subsys/bluetooth/controller/CMakeFiles/subsys__bluetooth__controller.dir/ll_sw/ull_conn.c.obj] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:3125: zephyr/subsys/bluetooth/controller/CMakeFiles/subsys__bluetooth__controller.dir/all] Error 2
gmake: *** [Makefile:104: all] Error 2
```

--- a/subsys/bluetooth/controller/ll_sw/ull_conn.c
+++ b/subsys/bluetooth/controller/ll_sw/ull_conn.c
@@ -6327,9 +6327,9 @@ static uint8_t force_md_cnt_calc(struct lll_conn *lll_conn, uint32_t tx_rate)
 	phy = PHY_1M;
 #endif /* !CONFIG_BT_CTLR_PHY */
 
-	time_incoming = (CONFIG_BT_CTLR_DATA_LENGTH_MAX << 3) *
+	time_incoming = (LL_LENGTH_OCTETS_RX_MAX << 3) *
 			1000000UL / tx_rate;
-	time_outgoing = PKT_DC_US(CONFIG_BT_CTLR_DATA_LENGTH_MAX,
+	time_outgoing = PKT_DC_US(LL_LENGTH_OCTETS_RX_MAX,
 				  (PDU_MIC_SIZE * lll_conn->enc_tx), phy) +
 			PKT_DC_US(0U, 0U, phy) +
 			(EVENT_IFS_US << 1);

